### ISSUE TITLE
Userbots and embedded messages

### ISSUE THREAD

**Issue by V4NSH4J** [Merubokkusu/Discord-S.C.U.M#311]
Have there been some recent changes disallowing userbots to send embedded messages? Is it still possible for userbots to send embedded messages?

**Comment by FuckingToasters** [comment]
> Have there been some recent changes disallowing userbots to send embedded messages? Is it still possible for userbots to send embedded messages?

Today i wanted to get back into programming a selfbot but then i noticed an ` HTTPException: 400 Bad Request` with the reason that it cant send empty messages however the message was not empty but a embed instead. I also heard from others that embeds arre patched on userbots. Hope there is a workaround

**Comment by OrionCodeDev** [comment]
Yeah same here, embed seems be patched, but we can do something in this case?

**Comment by OrionCodeDev** [comment]
https://www.reddit.com/r/Discord_selfbots/comments/sa0hc2/discord_embeds_patched/

```
my workaround is really crappy, but you could always just send the responses through a webhook

mine mentions you in the server it's in, and sends the response

i archived the project, but you can check here for my (python) webhook response functions
```
https://github.com/13-05/discord.RED/blob/main/selfbot_red.py#L122

maybe it's can little help

**Comment by FuckingToasters** [comment]
> https://www.reddit.com/r/Discord_selfbots/comments/sa0hc2/discord_embeds_patched/
> 
> ```
> my workaround is really crappy, but you could always just send the responses through a webhook
> 
> mine mentions you in the server it's in, and sends the response
> 
> i archived the project, but you can check here for my (python) webhook response functions
> ```
> 
> https://github.com/13-05/discord.RED/blob/main/selfbot_red.py#L122
> 
> maybe it's can little help

Well reading this seem to work however i not tried that solution so far but defently will check it out.
However i think even if this works you sometimes what to use the bot on servers you dont have the permission to use a webhook and still want to send some embes there. you could make it so the response is sent to a webhook on a own testingserver but that make not much sense i guess.
